content/blog/2016-01-26-one-year-of-open-prometheus-development.md

```diff
@@ -0,0 +1,148 @@
+[EC2](http://aws.amazon.com/ec2/).
+
+Shortly after making more noise about Prometheus, we also found one contributor
+([Fabian](https://github.com/fabxc)) so outstanding that he ended up joining
+SoundCloud to work on Prometheus. He has since become the most active developer
+on the project and we have him to thank for major new features such as
+generalized service discovery support, runtime-reloadable configurations, new
+powerful query language features, a custom-built query parser, and so much
+more. He is currently working on the new beta rewrite of the
+[Alertmanager](https://github.com/prometheus/alertmanager).
+
+Finally, we have been honored to be recognized and adopted by major players in
+the industry. [Google](https://www.google.com) is now instrumenting its open-source
+container management system [Kubernetes](http://kubernetes.io/) natively with
+Prometheus metrics. [CoreOS](https://coreos.com/) is picking it up for
+[etcd](https://coreos.com/etcd/)'s monitoring as well. [DigitalOcean](https://www.digitalocean.com/) is betting on Prometheus for their
+internal monitoring. By now, the list of companies using Prometheus in one way
+or another has become too long to mention: [Google](https://www.google.com),
+[CoreOS](https://coreos.com/), [Docker](https://docker.com),
+[Boxever](http://www.boxever.com/),
+[DigitalOcean](https://www.digitalocean.com/), [Financial Times](http://www.ft.com/),
+[Improbable](http://improbable.io/), [KPMG](https://kpmg.com), and many more.
+Even the world's largest digital festival,
+[DreamHack](https://www.dreamhack.se), has [used
+Prometheus](http://prometheus.io/blog/2015/06/24/monitoring-dreamhack/) to keep
+tabs on their network infrastructure in 2015, and
+[FOSDEM](https://fosdem.org/2016/) will do so in 2016.
+
+The widely popular dashboard builder [Grafana](http://grafana.org/) also added
+native Prometheus backend support in [version
+2.5](http://grafana.org/blog/2015/10/28/Grafana-2-5-Released.html). Since
+people all around the world are already using and loving Grafana, we are going
+to focus on improving Grafana's Prometheus integration and will invest
+less energy in our own dashboard builder
+[PromDash](https://github.com/prometheus/promdash) in the future.
+
+With the Prometheus ecosystem continuing to grow, the first users have started
+asking about commercial support. While Prometheus will always remain an
+independent open source project, one of our core contributors ([Brian
+Brazil](https://github.com/brian-brazil)) has recently founded his own company,
+[Robust Perception](http://www.robustperception.io/), which provides support
+and consulting services around Prometheus and monitoring in general.
+
+On a lighter note, 2015 has also been the year in which Prometheus's query
+language was proven to be Turing complete by Brian by implementing
+[Conway's Game of Life in PromQL](http://www.robustperception.io/conways-life-in-prometheus/).
```

Comment:
On a lighter note, 2015 has also been the year in which Brian proved Prometheus's query language to be Turing complete by implementing ...

(No passive voice, no double "by".)